orga/tests/integration/components/ui/empty-state_test.js

```diff
@@ -13,10 +13,10 @@ module('Integration | Component | Ui::EmptyState', function (hooks) {
     this.set('actionText', 'L’administrateur doit importer la base élèves en cliquant sur le bouton importer.');
 
     //when
-    await render(hbs`<Ui::EmptyState @infoText={{this.infoText}} @actionText={{this.actionText}} />`);
+    const screen = await render(hbs`<Ui::EmptyState @infoText={{this.infoText}} @actionText={{this.actionText}} />`);
 
     //then
-    assert.contains("Aucun participant pour l'instant !");
-    assert.contains('L’administrateur doit importer la base élèves en cliquant sur le bouton importer.');
+    assert.ok(screen.getByText("Aucun participant pour l'instant !"));
+    assert.ok(screen.getByText('L’administrateur doit importer la base élèves en cliquant sur le bouton importer.'));
```

Comment:
**suggestion**: Utiliser la clé de trad

Reply:
Ce sont des textes définit juste au dessus avec des `this.set` alors, malheureusement il n'y a pas de traduction